platform-api/src/main/java/io/datacater/core/deployment/StaticConfig.java

```diff
@@ -130,7 +124,7 @@ private EnvironmentVariables() {}
     static final String PYTHON_RUNNER_IMAGE_TAG =
         ConfigProvider.getConfig()
             .getOptionalValue("datacater.pythonrunner.image.version", String.class)
-            .orElse("alpha-20221117");
+            .orElse("nightly");
```

Comment:
[For other image versions](https://github.com/DataCater/datacater/blob/main/platform-api/src/main/java/io/datacater/core/deployment/StaticConfig.java#L95), we point to the latest stable release. Shall we do the same here?